core/src/test/scala/unit/kafka/server/AbstractFetcherThreadTest.scala

```diff
@@ -123,4 +125,97 @@ class AbstractFetcherThreadTest {
       new DummyFetchRequest(partitionMap.map { case (k, v) => (k, v.offset) }.toMap)
   }
 
+
+  @Test
+  def testFetchRequestCorruptedMessageException() {
+    val partition = new TopicPartition("topic", 0)
+    val fetcherThread = new TestFetcherThreadForCorruptedMessage("test", "client",
+      new BrokerEndPoint(0, "localhost", 9092), 100)
```

Comment:
This will make the test wait for 100 ms. It would be good if we can avoid adding to the already long unit test time. Is it possible to use `MockTime` class for the test?